pvlib/pvsystem.py

```diff
@@ -2508,7 +2520,8 @@ def max_power_point(photocurrent, saturation_current, resistance_series,
     """
     i_mp, v_mp, p_mp = _singlediode.bishop88_mpp(
         photocurrent, saturation_current, resistance_series,
-        resistance_shunt, nNsVth, method=method.lower()
+        resistance_shunt, nNsVth, d2mutau=0, NsVbi=np.Inf,
```

Comment:
Is it an oversight that these parameters are hardcoded with default values here instead of passing through the arguments to this function?

Reply:
I think it’s unintentional. Look at the git blame for these lines. When bishop was first added, we included the capability to model thin films but we didn’t fully implement passing the arguments and docstring in all the single diode & `pvsystem` functions & classes, instead we opened #517  and then @adriesse opened this PR but I think we overlooked this line sorry. Oops!

Reply:
I probably copy/pasted from the function signature.  :(

Reply:
@kandersolar & @adriesse addressed in https://github.com/pvlib/pvlib-python/pull/1733 - btw: the recombination losses were only implemented in `pvsystem` for mpp, because the other `*_from_*()` functions have an option for `'lambertw'` which doesn't currently handle recombination losses. Interestingly, voltages approaching Voc are more likely to be affected by recombination losses than at Vmp, so it would be nice to implement them for `*_from_*()` at some point, but it will require some creative branching to handle the `"method"` arg.